tests/format/js/destructuring-assignment/jsfmt.spec.js

```diff
@@ -0,0 +1 @@
+run_spec(import.meta, ["babel", "flow", "typescript"], { printWidth: 20 });
```

Comment:
Please add tests with default options:

```suggestion
run_spec(import.meta, ["babel", "flow", "typescript"]);
run_spec(import.meta, ["babel", "flow", "typescript"], { printWidth: 20 });
```